modules/markup/html.go

```diff
@@ -305,18 +305,12 @@ func postProcess(ctx *RenderContext, procs []processor, input io.Reader, output
 		return err
 	}
 
-	res := bytes.NewBuffer(make([]byte, 0, len(rawHTML)+50))
-	// prepend "<html><body>"
-	_, _ = res.WriteString("<html><body>")
-
-	// Strip out nuls - they're always invalid
-	_, _ = res.Write(tagCleaner.ReplaceAll([]byte(nulCleaner.Replace(string(rawHTML))), []byte("&lt;$1")))
-
-	// close the tags
-	_, _ = res.WriteString("</body></html>")
-
 	// parse the HTML
-	node, err := html.Parse(res)
+	node, err := html.Parse(io.MultiReader(
+		strings.NewReader("<html><body>"),
+		bytes.NewReader(tagCleaner.ReplaceAll([]byte(nulCleaner.Replace(string(rawHTML))), []byte("&lt;$1"))),
```

Comment:
Can you keep the comments from before?

Reply:
At least `// Strip out nuls - they're always invalid`, the others are fairly irrelevant.

Reply:
Done.